runhouse/main.py

```diff
@@ -452,23 +454,23 @@ def status(
             )
             raise typer.Exit(1)
 
-    # case we are inside the cluster
-    if cluster_or_local != "file":
-        # If we are on the cluster load status directly from the object store
-        cluster_status: dict = dict(obj_store.status())
-        cluster_config = copy.deepcopy(cluster_status.get("cluster_config"))
-        current_cluster: Cluster = Cluster.from_config(cluster_config)
-        return _print_status(cluster_status, current_cluster)
-
-    if cluster_name is None:
+    if cluster_or_local == "file" and not cluster_name:
         # If running outside the cluster must specify a cluster name
         console.print("Missing argument `cluster_name`.")
         return
 
+    # case we are inside the cluster
+    elif cluster_or_local != "file":
```

Comment:
i think we can delete this entire elif statement? bc the call inside the try / except handles the case where we are inside the cluster

Reply:
yeah, i have it fixed, will push in a few

Reply:
yeah but we need to create a cluster instance to call `current_cluster.status()`, which handles the case where we are inside the cluster. In order to create the cluster instance i need its name, and when we are running `runhouse status` inside the cluster we allow the user not to pass the cluster name. 

So this elif clause is still relevant unfortunately.

Reply:
can we avoid parsing the file manually tho? if we're on the cluster `rh.here.name` should suffice